# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/pagination/reftest.list
# Commit: f733b37f9626
# Full Hash: f733b37f9626548ae83564a3fd43ddfa972977d9
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2021-04-22 21:31:57
# Regressor Bug: 1702228
# File Overlap Count: 1
# Description:
#   Bug 1702228 - Adjust reftests. r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112970
# ==============================================================================

diff -r 0919633e25e9 -r f733b37f9626 layout/reftests/pagination/reftest.list
--- a/layout/reftests/pagination/reftest.list	Thu Apr 22 10:34:55 2021 +0000
+++ b/layout/reftests/pagination/reftest.list	Thu Apr 22 10:34:55 2021 +0000
@@ -179,7 +179,7 @@
 == bfc-007.html bfc-006-ref.html
 == bfc-008.html bfc-006-ref.html
 
-pref(layout.display-list.improve-fragmentation,true) fuzzy-if(cocoaWidget&&swgl,1-1,21-21) == inline-block-frag-simple-1.html inline-block-frag-simple-1-ref.html
+pref(layout.display-list.improve-fragmentation,true) fuzzy-if(swgl,0-1,0-200) == inline-block-frag-simple-1.html inline-block-frag-simple-1-ref.html
 pref(layout.display-list.improve-fragmentation,true) fuzzy-if(cocoaWidget&&swgl,0-1,0-21) == inline-block-frag-simple-2.html inline-block-frag-simple-2-ref.html
 pref(layout.display-list.improve-fragmentation,true) == inline-block-frag-offset-1.html inline-block-frag-offset-1-ref.html
 pref(layout.display-list.improve-fragmentation,true) == inline-block-frag-offset-2.html inline-block-frag-offset-2-ref.html